app/templates/tasklist.html

```diff
@@ -9,13 +9,28 @@ <h1 class="govuk-heading-xl">
       Round: {{application_meta_data["round"]|snake_case_to_human}}
     </h1>
     {% if not application_response["status"] == application_meta_data["submitted_status"] %}
-    <h2 class="govuk-body govuk-!-font-weight-bold">
-      {% if application_meta_data["number_of_completed_sections"] == application_meta_data["number_of_sections"] %}
-       All sections complete
-      {% else %}
-       In progress
+
+    <p class="govuk-body-l govuk-!-font-weight-bold">
+      Status: 
+      {% set completed_count = application_meta_data["number_of_completed_sections"] %}
+      {% set section_count = application_meta_data["number_of_sections"] %}
+      {% if completed_count == 0 %}
+      <strong class="govuk-tag govuk-tag--grey">
+        Not Started
+      </strong>
+      {% endif %}
+      {% if completed_count > 0 and completed_count < section_count %}
+      <strong class="govuk-tag govuk-tag--blue">
+        In Progress
+      </strong>
       {% endif %}
-    </h2>
+      {% if completed_count == section_count %}
+      <strong class="govuk-tag govuk-tag--green">
+        Complete
+      </strong>
+      {% endif %}
```

Comment:
Could we use `elif` here as these should be exclusive? e.g. https://jinja.palletsprojects.com/en/3.0.x/templates/#if

Reply:
good point - implementing